### ISSUE TITLE
Error while updating the slider value:  Cannot read property 'toFixed' of undefined

### ISSUE THREAD

**Issue by rajeshwarpatlolla** [react-component/slider#456]
There is an error in line#97 in `lib > utils.js`
return step === null ? closestPoint : parseFloat(closestPoint.toFixed(getPrecision(step)));

Please fix this error.

![screen shot 2018-07-30 at 1 15 20 pm](https://user-images.githubusercontent.com/9896335/43384131-b4fe8dc8-93fa-11e8-961c-799014270edc.png)

**Comment by Sherlock92** [comment]
The issue occurs when no width is specified on the slider when in horizontal mode, or when no height is specified in vertical mode.

**Comment by rajeshwarpatlolla** [comment]
I don't think that is related to width and height. I tried specifying width and height but no use.

**Comment by nickrobillard** [comment]
I'm getting this exact error but in IE11 only - Chrome/FF are fine. Any update here?
